src/main/java/uk/gov/digital/ho/hocs/document/DocumentDataService.java

```diff
@@ -103,7 +102,12 @@ public S3Document getDocumentPdf(UUID documentUUID) {
         DocumentData documentData = getDocumentData(documentUUID);
         try{
             log.debug("Getting Document PDF: {}", documentUUID);
-            return s3DocumentService.getFileFromTrustedS3(documentData.getPdfLink());
+            String pdfLink = documentData.getPdfLink();
```

Comment:
We should not replace the return type with non pfd format, let's just throw the exception here.
Also documentDto should now include the flag 'hasPdf" in order for the client to know if this call is valid. 
Thanks